ts/README.md

```diff
@@ -113,40 +113,43 @@ function a(data?: Array<(number|undefined)>, dOffset?: number): number {
 ```
 
 
-### Type union instead of new interface
 
-Sometimes you just need a pure merge of several classes or interfaces as a new
-type. Instead of creating an empty interface that extends them, you create a
-type union. This also works as an inline type.
+Good to Know
+------------
 
-*Incorrect:* 
-```ts
-interface A {
-    x: number;
-}
-interface B {
-    x2: number
-}
-interface C extends A, B {
-}
-```  
-*Correct:*
-```ts
-interface A {
-    x: number;
-}
-interface B {
-    x2: number
-}
-type C = (A&B);
-let example1: C = { x: 1, x2: 2 },
-    example2: (A&B) = example1;
-```
 
+### Make use of common ES6 features
 
+TypeScript does transpile most common ES6 features down to ES5-capable browsers
+like Internet Explorer 9+, Chrome 5+, Firefox 4+, Safari 5+. So you can make use
+of them:
 
-Good to Know
-------------
+* Arrow functions (without own `this` scope):
+  `(a, b) => {console.log('yeah')};`
+
+* Constants and limited variables:
+  `const yes: boolean = true; let degree: number = 180;`
+
+* Default parameter values:
+  `function (required: number, optional: string = ''): void;`
+
+* Destructing arrays:
+  `const sentence = ['Hello', 'world']; let [greeting, pronoun] = test;`
```

Comment:
Small typo with `let [greeting, pronoun] = test;`, should be `let [greeting, pronoun] = sentence;`

Reply:
Thanks, fixed.